### DISCUSSION TITLE
Works on windows?

### DISCUSSION THREAD

**Discussion by marian-bucur** [avocado-framework/avocado#4433] [Q&A]
Hi, I'm new here. This project runs on Windows? I'm looking for a way to run programs to test my software on Windows and Linux. Thanks.

**Comment by beraldoleal** [comment]
Hi @marian-bucur,

Unfortunately, we don't provide official support to Windows machines. "In theory", depending on the runner and the plugins that you are using,  you could. But probably you will hit some walls.

**Reply by marian-bucur** [reply]
Thank you. I hope I'm going to use it and contribute fixes.
